functions/habapp.bash

```diff
@@ -49,8 +49,7 @@ habapp_setup() {
 
   echo -n "$(timestamp) [openHABian] Creating HABApp configuration folder ... "
   if ! cond_redirect mkdir -p "$configFolder"; then echo "FAILED"; return 1; fi
-  if ! cond_redirect chown -R "openhab:${username:-openhabian}" "$configFolder"; then echo "FAILED"; return 1; fi
-  if cond_redirect fix_permissions "$configFolder" 775 775; then echo "OK"; else echo "FAILED"; return 1; fi
+  if cond_redirect fix_permissions "$configFolder" "openhab:${username:-openhabian}" 775 775; then echo "OK"; else echo "FAILED"; return 1; fi
```

Comment:
Does the 755 have to be "755" ?

Reply:
Yes it's recursive. 755 is probably ok, too, and a little bit safer.

Reply:
Perfect - Then I think this looks good.